.github/workflows/artifacts-build.yml

```diff
@@ -308,7 +308,7 @@ jobs:
   bench-memory:
     # This is too expensive to run on every push, so only run it on main.
     # When running on a PR, comment this out and set the BASELINE variable below to the baseline commit.
-    if: github.ref == 'refs/heads/main' && github.repository == 'unicode-org/icu4x'
+    # if: github.ref == 'refs/heads/main' && github.repository == 'unicode-org/icu4x'
```

Comment:
Note that this won't work on other forks because of the Workload Identity Provider; is that what you want?